1209/F16A/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Cougar TQS adapter
+owner: uri_ba
+license: MIT
```

Comment:
licence is in the project, "licence.txt" in the main folder, and it is applies to the software as it is the original framework license. I've added an additional license file for the HW. and properly indicated it in the project README.

I was not aware that a board design was required, I had only created a schematics as it was intended to be hand soldered. I will create one and commit it later this week.

Thank you,
Uri